common/djangoapps/student/models.py

```diff
@@ -179,6 +179,17 @@ class UserStanding(models.Model):
     standing_last_changed_at = models.DateTimeField(auto_now=True)
 
 
+class UserStandingConfig(ConfigurationModel):
+    disabled = models.TextField(
+        help_text="White-space separated list of usernames of users who should be disabled",
```

Comment:
Addition is only going to be happening from the admin interface. But yes, as this grows to longer than a small handful, I agree, this interface isn't going to be great. I'm honestly not sure how to best balance the needs of not spamming the cache (by creating an entry for every user) vs not having a single giant list. I'm inclined for the moment to simply lean towards what's simple for the small list we have, and tackle the large list later.